src/lib/components/forms/file-input.stories.ts

```diff
@@ -0,0 +1,11 @@
+import FileInput from './file-input.svelte';
+
+export default {
+  title: 'Atoms/Forms/File Input',
+  component: FileInput,
+  parameters: {
+    layout: 'centered'
+  }
+};
+
+export const TestFileInput = {};
```

Comment:
```suggestion
export default {
  title: 'Atoms/Forms',
  component: FileInput,
  parameters: {
    layout: 'centered'
  }
};

export const File_Input = {};
```
When you only have one example you don't need the extra "folder"